README.md

```diff
@@ -26,6 +26,7 @@ Install `npm`, `python 3.11`, and [`pipenv`](https://pipenv.pypa.io/en/latest/).
 
 Some things (e.g. logging) require a database connection to work correctly. To make this easier, there is a script to set one up locally via Docker. To get this working:
 
+* Install mysql on your local machine
```

Comment:
you should probably be able to just use the docker image now. The previous mysql-python library did need mysql installed locally, but it's been changed to a pure python one